### DISCUSSION TITLE
shell介紹

### DISCUSSION THREAD

**Discussion by minniewei** [1101-GS4719-C/Python-Community-GS4719-C#539] [General]
最近看到一篇shell的文章並得知原來cmd或檔案總管都是一種shell。藉由shell我們能與作業系統的kernel進行溝通，而核心再跟各項硬體設備連結，所以了解shell的語法或架構能幫助我們解決許多電腦問題，以下是關於shell的一些基本介紹:
https://ithelp.ithome.com.tw/articles/10207473
各位如果有在使用linux也可透過下面的文章了解如何查詢shell的版本及種類。
https://dotblogs.com.tw/dc690216/2011/07/08/31303

**Comment by jason79461385** [comment]
感謝分享  話說之前做專題時還有因為shell版本導致script檔run不出來的情況呢 哈哈哈哈
